spec/services/transcript_service_spec.rb

```diff
@@ -113,4 +113,31 @@
       end
     end
   end
+
+  describe ".find" do
+    # rubocop:disable Rails/DynamicFindBy
+    # This is the method in service
+    subject(:searching_transcript) do
+      described_class.find_by_uid(transcript.uid)
+    end
+    # rubocop:enable Rails/DynamicFindBy
+
+    let(:transcript) { FactoryBot.create :transcript, publish: publish }
```

Comment:
@sameera207 You could also include the `FactoryBot` helpers to `rails_helper` so you can write `create/build` directly. Or was this a preference?

Reply:
@adimasuhid , thanks, No preference, I should start using just `create` :)